lockbox-ios/Presenter/WelcomePresenter.swift

```diff
@@ -59,10 +73,16 @@ class WelcomePresenter {
                     .disposed(by: self.disposeBag)
 
             Observable.combineLatest(lockedObservable, biometricsObservable)
-                    .map { LockedEnabled(appLocked: $0.0, biometricsEnabled: $0.1) }
+                    .map {
+                        LockedEnabled(appLocked: $0.0, biometricsEnabled: $0.1)
```

Comment:
@sashei I think I was able to bypass the "Sign Out" (from Settings) lock:

1. open Settings > Sign Out
2. see "Welcome" screen showing locked, as expected ✅ 
3. Close and re-open app
4. See "Welcome" screen showing locked, as expected ✅ 
5. Force close / kill app and re-open
6. See the item list view, not prompted by "Welcome" to unlock ⚠️

Reply:
oooo good catch Devin, I think this is actually in the `RootPresenter`.... I will address.